wiki/Announcements/Contests/Monthly_Fanart_Contest/en.md

```diff
@@ -8,36 +8,48 @@ All offical current and completed monthly fanart contests are listed here from o
 
 - Results: [First Official osu! Fanart Contest Results!](https://osu.ppy.sh/news/86373762103)
 - See the Facebook album: [First Official osu! Fanart Contest Results!](https://www.facebook.com/pg/osugame/photos/?tab=album&album_id=10152186520298282)
-
-To see the fanart selections for this contest, see [Pippi, the osu! mascot!](Pippi,_the_osu!_mascot!).
+- Fanart selections, see [Pippi, the osu! mascot!](Pippi,_the_osu!_mascot!).
 
 ### osu! Around the World!
 
 - Results: [Second Official osu! Fanart Contest Results!](https://osu.ppy.sh/news/90764226618)
 - See the Facebook album: [osu! Fanart Contest 2!](https://www.facebook.com/pg/osugame/photos/?tab=album&album_id=10152291754173282)
-
-To see the fanart selections for this contest, see [osu! Around the World!](osu!_Around_the_World!).
+- Fanart selections, see [osu! Around the World!](osu!_Around_the_World!).
 
 ### So Many Stickers!
 
 - Results: [Third Official osu! Fanart Contest Results!](https://osu.ppy.sh/news/99609478178)
 - See the Facebook album: [osu! Fanart Contest 3!](https://www.facebook.com/pg/osugame/photos/?tab=album&album_id=10152514671088282)
+- Fanart selections, see [So Many Stickers!](So_Many_Stickers!).
+
+<!-- where are the other two? -->
```

Comment:
Why is this here? No need to add this comment. Make an issue about it, not a note here.